### PR TITLE
Fix for add/delete/update bookmarks

### PR DESCRIPTION
**Pull request by elland**
Code was not properly accounting for positioning/sorting by date.

Comments should explain quite well how and why the code is the way it is, and which use cases it's not currently accounting for. Please let me know if something is not clear, missing, etc.

### CODE REVIEW

**Comment by elland** [comment]
cc @3lvis

File: Library/Fetchers/Fetcher.swift [hunk 1]
```diff
@@ -40,10 +40,10 @@ class Fetcher {
         }
     }
 
-    func computePath(for path: String, with parameters: [String: String]? = nil) -> String {
+    func computePath(for path: String, with parameters: [String: AnyHashable]? = nil) -> String {
         var parametersString = ""
         if let parameters = parameters {
-            parametersString = parameters.map { name, value in "\(name.urlEncoded)=\(value.urlEncoded)" }.joined(separator: "&")
+            parametersString = parameters.map { name, value in "\(name)=\(value)" }.joined(separator: "&")
```
Comment (3lvis) [inline Library/Fetchers/Fetcher.swift:53]: Maybe you want this https://github.com/3lvis/Networking/blob/master/README.md#percent-encoding

Comment (elland) [reply Library/Fetchers/Fetcher.swift:53]: No, I don't think so. Unless Networking exposes a method to convert a dictionary into URL form-encoded strings that also accepts non-string values, which it doesn't, AFAIK. 💭

**Comment by elland** [comment]
What do you think? 🤔

File: Library/Fetchers/Fetcher.swift (hunk 1 above)
Comment (3lvis) [reply Library/Fetchers/Fetcher.swift:53]: I don't follow.

Comment (elland) [reply Library/Fetchers/Fetcher.swift:53]: The method you pointed to doesn't satisfy the needs of this particular use case.

Comment (3lvis) [reply Library/Fetchers/Fetcher.swift:53]: Isn't this what yo want?

https://github.com/3lvis/Networking/blob/master/Tests/Dictionary%2BFormURLEncodedTests.swift

Comment (elland) [reply Library/Fetchers/Fetcher.swift:53]: Oh, some typos there. Updated the comment.

Comment (3lvis) [reply Library/Fetchers/Fetcher.swift:53]: I understand now.

**Comment by 3lvis** [comment]
Shipppit

**Pull request merged**